README.md

```diff
@@ -1,9 +1,9 @@
 # 🐉 Rust GPU
 
-[![Contributor Covenant](https://img.shields.io/badge/contributor%20covenant-v1.4%20adopted-ff69b4.svg)](CODE_OF_CONDUCT.md)
-[![dependency status](https://deps.rs/repo/github/EmbarkStudios/rust-gpu/status.svg)](https://deps.rs/repo/github/EmbarkStudios/rust-gpu)
-[![Embark](https://img.shields.io/badge/embark-open%20source-blueviolet.svg)](http://embark.dev)
+[![Embark](https://img.shields.io/badge/embark-open%20source-blueviolet.svg)](http://embark.games)
```

Comment:
shouldn't this stay `embark.dev` rather than `embark.games`? I guess this is a question for @repi or someone.

Reply:
It should, this is a copy/paste mistake from me

Reply:
Yes should be embark.dev, think it is in the other repos and these badge updates?

Reply:
Yes I copied `.games` from somewhere and had to correct a couple after I realised. I missed this one when checking as it wasn't grouped in the same list in the issue. 

The top of my screen was where the red line was, oops.

![image](https://user-images.githubusercontent.com/6134406/97422898-5adf1680-1906-11eb-8bff-854d316a6d3d.png)